src/components/AdminSettings/SignalingServer.vue

```diff
@@ -38,17 +38,23 @@
 			@change="updateVerify">
 		<label :for="'verify' + index">{{ t('spreed', 'Validate SSL certificate') }}</label>
 
-		<a v-show="!loading"
-			v-tooltip.auto="t('spreed', 'Delete this server')"
```

Comment:
Just a hint: `ButtonVue` has no tooltip by default, so if you want to keep it, you need to add it manually.

Reply:
Should be fine for now if it is the accepted default behaviour. The icon should indicate it pretty obvious and screenreaders have the aria-label